### PR TITLE
Fix type error in DropPointsByClass

### PR DESCRIPTION
**Pull request by leavauchier**
A `TypeError: object of type 'numpy.int64' has no len()` was raised in some specific use case when using DropPointsByClass with batches containing a single point (the numpy array was transformed to an integer)

### CODE REVIEW

File: tests/myria3d/models/test_model.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import hydra
+from pytorch_lightning import LightningDataModule
+
+from myria3d.models.model import Model
+
+
+def test_model_get_batch_tensor_by_enumeration():
+    with hydra.initialize(config_path="./../../../trained_model_assets/", job_name="config"):
+        config = hydra.compose(
+            config_name="proto151_V2.0_epoch_100_Myria3DV3.1.0_predict_config_V3.7.0",
```
Comment (CharlesGaydon) [inline tests/myria3d/models/test_model.py:10]: This config_name is hardcoded. Can it be defined imported (I think it is defined in run.py or train.py).

**Review by CharlesGaydon — CHANGES_REQUESTED** [review]
Small improvement could be made by referencing config file via a variable. Apart from that looks good to me :)

File: tests/myria3d/pctl/transforms/test_transforms.py [hunk 2]
```diff
@@ -12,10 +12,12 @@ def test_TargetTransform_with_valid_config():
...
     tt = TargetTransform(classification_preprocessing_dict, classification_dict)
-
     y = np.array([1, 1, 2, 2, 6, 6])
-    data = torch_geometric.data.Data(x=None, y=y)
-    assert np.array_equal(tt(data).y, np.array([0, 0, 0, 0, 1, 1]))
+    idx = np.arange(6)
+    data = torch_geometric.data.Data(x=None, y=y, idx_in_original_cloud=idx)
+    out_data = tt(data)
+    assert np.array_equal(out_data.y, np.array([0, 0, 0, 0, 1, 1]))
+    assert np.array_equal(out_data.idx_in_original_cloud, idx)
```
[the remark this answers is not in the document]
Comment (leavauchier) [reply tests/myria3d/pctl/transforms/test_transforms.py:20]: yes, after all I'm not sure it is useful, but I did it to be consistent with the test I added for DropPointsByClass

File: tests/myria3d/models/test_model.py (hunk 1 above)
Comment (leavauchier) [reply tests/myria3d/models/test_model.py:10]: Thanks, looks a good idea, but I'm not sure it is easy to get DEFAULT_CONFIG_FILE from `run.py` as it is not in the myria3d python module (hence I suppose it is not intended to be imported)

Comment (CharlesGaydon) [reply tests/myria3d/models/test_model.py:10]: Inded, but consequently I do not think it is supposed to be used for testing. 
In test I usually used function `make_default_hydra_cfg` from `conftest.py`, to always test for the default configuration.

Comment (leavauchier) [reply tests/myria3d/models/test_model.py]: I finally managed to find how to use it (the issue was that datamodule was using hydra:runtime.cwd, so I had to override the parameter that required it

**Review by CharlesGaydon — APPROVED** [review]

**Comment by CharlesGaydon** [comment]
LGTM !

**Pull request merged**